src/Symfony/Component/Lock/Tests/Store/ZookeeperStoreTest.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Lock\Tests\Store;
+
+/*
+ *
+ * @author Ganesh Chandrasekaran <gchandrasekaran@wayfair.com>
+ *
+ * @requires extension zookeeper
```

Comment:
I need help with this - Does this make the tests skip if the environment doesn't have a Zookeeper extension installed?

Reply:
Probably needs to be a class level annotation, not a generic comment in the file

Reply:
This was in wrong place by mistake. Moved.